scripts/upload_changed.py

```diff
@@ -28,13 +29,21 @@ def main(typeshed_dir: str, commit: str, dry_run: bool = False) -> None:
         if dry_run:
             print(f"Would upload: {distribution}, increment {increment}")
             continue
+        for dependency in build_wheel.read_matadata(
```

Comment:
Share this code? It seems repeated.

Reply:
Yeah, I tried, but because of one line in the middle that is different it doesn't look worth it.